match/spec/storage/gitlab/client_spec.rb

```diff
@@ -1,9 +1,12 @@
 describe Match do
   describe Match::Storage::GitLab::Client do
-    subject { described_class.new(
-      api_v4_url: 'https://gitlab.example.com/api/v4', 
-      project_id: 'sample/project',
-      private_token: 'abc123') }
+    subject {
+      described_class.new(
+        api_v4_url: 'https://gitlab.example.com/api/v4',
+        project_id: 'sample/project',
+        private_token: 'abc123'
+    )
```

Comment:
Should this one be indented?

Reply:
It probably should 😅

Reply:
But it doesn't make CI angry and I don't feel like waiting for CI to run again tonight so I'll let that one slide for now 😉